web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -60,11 +62,243 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
     $scope.loadPage(false);
   };
 
+  $scope.selectedItems = [];
+
+  $scope.showActionBar = false;
+
+  $scope.$watch('selectedItems', function (newValue) {
+    $scope.badgeSelection = {};
+    $scope.awardBadgeButtonEnabled = false;
+  });
+
+  $scope.$watch('selectedItems.length', function (newValue) {
+    $scope.badgeSelection = {};
+    $scope.awardBadgeButtonEnabled = false;
+    $scope.showActionBar = newValue > 0;
+  });
+
+  $scope.adultRoles = [
+    {
+      name: 'Parent/Guardian',
+      userType: {
+        name: 'parent-guardian'
+      },
+      image: userUtils.defaultAvatar('parent-guardian'),
+      approvalRequired: false,
+      text: $translate.instant('Parent/Guardian - Parents/kids are super important in CoderDojo as they encourage and inspire CoderDojo Ninjas on a daily basis! Parents can sign up on the platform to register their kids (both aged under 13 and over 13) for their Local Dojos events and so their kids can earn badges for their profile!'),
+      class: 'cd-parent',
+      buttonText: $translate.instant('Change Role')
+    },
+    {
+      name: 'Volunteer',
+      userType: {
+        name:  'mentor'
+      },
+      approvalRequired: true,
+      image: userUtils.defaultAvatar('mentor'),
+      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      class: 'cd-volunteer',
+      buttonText: $translate.instant('Change Role')
+    },
+    {
+      name: 'Champion',
+      userType: {
+        name:  'champion'
+      },
+      approvalRequired: true,
+      image: userUtils.defaultAvatar('champion'),
+      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      class: 'cd-champion',
+      buttonText: $translate.instant('Change Role')
+    }
+  ];
+
+  $scope.youthRoles = [
+    {
+      name: 'Attendee',
+      image: userUtils.defaultAvatar('ninja'),
+      approvalRequired: false,
+      text: $translate.instant('Attendees at CoderDojo are young people under 18 who attend and make CoderDojo cool. Often known as Ninjas, CoderDojo attendees come to the Dojo to have fun and bring their awesome ideas to life.'),
+      class: 'cd-parent',
+      buttonText: $translate.instant('Change Role')
+    },
+    {
+      name: 'Youth Volunteer',
+      userType: {
+        name:  'mentor'
+      },
+      approvalRequired: true,
+      image: userUtils.defaultAvatar('mentor'),
+      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
```

Comment:
Whoops... Wrong line for youths. Instead here, align with @Wardormeur's PR on changing these.